common/app/implicits/FaciaContentFrontendHelpers.scala

```diff
@@ -22,6 +24,19 @@ object FaciaContentFrontendHelpers {
       imageOverride.orElse(defaultTrailPicture)
     }
 
+    def mainYouTubeMediaAtom: Option[MediaAtom] =
+      for {
+       main <- faciaContent.properties.maybeContent.map(_.fields.main)
+       atoms <-  faciaContent.properties.maybeContent.flatMap(_.atoms)
+       document <- Some(Jsoup.parse(main))
```

Comment:
OK I had a look at this library but it does not look trivial to integrate so I would rather ship this as a TODO. I have some WIP attempt at adding the `mainBlock` field in https://github.com/guardian/frontend/tree/playable-media-atoms-fronts however it has a runtime error.